jwst/ramp_fitting/ramp_fit.py

```diff
@@ -462,8 +462,7 @@ def ols_ramp_fit(model, buffsize, save_opt, readnoise_model, gain_model,
             if rhi > cubeshape[1]:
                 rhi = cubeshape[1]
 
-            # data_sect and gdq_sect are: [ groups, y, x ]
-            data_sect = model.get_section('data')[num_int, :, rlo:rhi, :]
```

Comment:
So is ``data_sect`` not needed anymore? Was flake8 complaining because it was unused?

Reply:
Yes, flake8 was complaining: " F841 local variable 'data_sect' is assigned to but never used."

Reply:
Odd why this wasn't getting picked up via our `flake8` checker in Travis CI?  I.e. it doesn't show up as a problem on `master` now.

Reply:
Oh, I see.  Because it is being ignored.

https://github.com/spacetelescope/jwst/blob/3619272747ef5a24eea898e4fd828868f6de6f0e/setup.cfg#L18-L28

If you want to do `flake8` cleanup, I would suggest removing the line above from `setup.cfg` and then dealing with the ~170 problems produced by that.  =)

Reply:
If you remove the `exclude` for `ramp_fitting` in `setup.cfg`, then that will keep it fixed into the future as well.

Eventually, once all our excludes are gone, we can put a pre-commit hook to do a `flake8` check.